### DISCUSSION TITLE
Could not connect to the remote site.

### DISCUSSION THREAD

**Discussion by r3dvorak** [akeeba/panopticon#677] [Q&A (Community Support)]
Hi,
on just one site out +100, I can't seem to connect. Always getting the same error 500. 
So far, I have emptied Cache, un- & reinstalled pkg_panopticon, recreated the Joomla API Token to no avail. 
The site itself throws no errors. This is a very old site that has been migrated many times, starting with Joomla 1.5. 
Any pointers?

**Troubleshooting information**
-
**Uh oh. Something is totally broken here…** 
_Process step: Authenticated access (can I get information out of the API?)_ 

HTTP Status | 500
HTTP Body | {"errors":[{"code":500,"title":"Internal server error"}]}
HTTP HEADERS
Date    Tue, 21 May 2024 07:52:13 GMT 
Server    Apache 
X-Powered-By    JoomlaAPI/1.0 
Expires    Wed, 17 Aug 2005 00:00:00 GMT 
Cache-Control    no-store, no-cache, must-revalidate, post-check=0, pre-check=0 
Pragma    no-cache 
Strict-Transport-Security    max-age=17171717 
X-Content-Type-Options    nosniff 
Last-Modified    Tue, 21 May 2024 07:52:13 GMT 
Vary    User-Agent
Connection    close 
Transfer-Encoding    chunked 
Content-Type    application/vnd.api+json; charset=utf-8 

It also claims that "You do not have Akeeba Backup Professional installed" -which is not true. But I would guess it can't get the information due tot the error 500.

Richard

**Comment by r3dvorak** [comment]
Update: I duplicated the site, reinstalled Joomla 5.1 Core files - now I get 
HTTP Status | 200
HTTP Body | <br />
<b>Deprecated</b>:  explode(): Passing null to parameter #2 ($string) of type string is deprecated in <b>/www/htdocs/w01880dd/fix.him-spvgg.de/libraries/joomlatools/component/koowa/translator/translator.php</b> on line <b>76</b><br />
<p><strong>RuntimeException</strong>: 'Headers already send (output started at /www/htdocs/w01880dd/fix.him-spvgg.de/libraries/joomlatools/component/koowa/translator/translator.php:76' thrown in <strong>/www/htdocs/w01880dd/fix.him-spvgg.de/libraries/joomlatools/library/dispatcher/response/transport/http.php</strong> on line <strong>56</strong></p><p>while handling exception</p><p><strong>Joomla\CMS\Access\Exception\AuthenticationFailed</strong>: '' thrown in <strong>/www/htdocs/w01880dd/fix.him-spvgg.de/libraries/src/Application/ApiApplication.php</strong> on line <strong>317</strong></p><h3>Stacktrace</h3><pre>#0 /www/htdocs/w01880dd/fix.him-spvgg.de/libraries/src/Application/ApiApplication.php(110): Joomla\CMS\Application\ApiApplication->route()
#1 /www/htdocs/w01880dd/fix.him-spvgg.de/libraries/src/Application/CMSApplication.php(306): Joomla\CMS\Application\ApiApplication->doExecute()
#2 /www/htdocs/w01880dd/fix.him-spvgg.de/api/includes/app.php(50): Joomla\CMS\Application\CMSApplication->execute()
#3 /www/htdocs/w01880dd/fix.him-spvgg.de/api/index.php(31): require_once('/www/htdocs/w01...')
#4 {main}</pre>

**Reply by brianteeman** [reply]
do you have koowa running on any of the other sites?

**Reply by r3dvorak** [reply]
Yes, without any problems, even on the same webserver. 
Nevertheless - I did uninstall Joomlatools/Koowa - but the site just won't connect.

**Comment by r3dvorak** [comment]
The culprit was Joomlatools Connect - after uninstalling it, the site was connected.

**Reply by r3dvorak** [reply]
It looks like the joomlatools / koowa framework causes an error 500  - the site is connected, but after reinstalling DOCman, I get:
Akeeba Backup is not linked.
There has been a communication error with the Akeeba Panopticon Connector on the remote site.
HTTP 500 Internal Server Error

**Reply by nikosdion** [reply]
You should contact JoomlaTools. I don't have a debug trace so I can't be absolutely certain, but from your description of the issue it looks like their system plugin might be conflicting with Joomla's API application. I wrote a blog post two years ago discussing how this kind of problem can be easily fixed: https://www.dionysopoulos.me/common-mistakes-writing-joomla-plugins.html see under "There is more to Joomla than the frontend and backend"
